integrationtests/supervisor/integrationtest_supervisor.py

```diff
@@ -5,23 +5,31 @@
 from google.protobuf.message import Message
 from integrationtests.utils import (
     DUMMY_CONFIG_FILE,
-    MNIST_CONFIG_FILE,
+    RHO_LOSS_CONFIG_FILE,
+    ImageDatasetHelper,
     TinyDatasetHelper,
     connect_to_server,
+    get_modyn_config,
     load_config_from_file,
 )
+from modyn.metadata_database.metadata_database_connection import MetadataDatabaseConnection
+from modyn.metadata_database.models import Pipeline, TrainedModel, Trigger
+from modyn.selector.internal.grpc.generated.selector_pb2 import GetSelectionStrategyRequest
+from modyn.selector.internal.grpc.generated.selector_pb2_grpc import SelectorStub
 from modyn.supervisor.internal.grpc.enums import PipelineStage, PipelineStatus
-from modyn.supervisor.internal.grpc.generated.supervisor_pb2 import GetPipelineStatusRequest
+from modyn.supervisor.internal.grpc.generated.supervisor_pb2 import GetPipelineStatusRequest, GetPipelineStatusResponse
 from modyn.supervisor.internal.grpc.generated.supervisor_pb2 import JsonString as SupervisorJsonString
 from modyn.supervisor.internal.grpc.generated.supervisor_pb2 import StartPipelineRequest
 from modyn.supervisor.internal.grpc.generated.supervisor_pb2_grpc import SupervisorStub
 
 POLL_INTERVAL = 1
+MODYN_CONFIG = get_modyn_config()
+IMAGE_DATASET_ID = "image_test_dataset"
 
 
-def wait_for_pipeline(pipeline_id: int) -> str:
+def wait_for_pipeline(pipeline_id: int) -> GetPipelineStatusResponse:
     req = GetPipelineStatusRequest(pipeline_id=pipeline_id)
-
+    supervisor = SupervisorStub(connect_to_server("supervisor"))
```

Comment:
previously supervisor variable is used implicitly. It is initialized in if __name__ == '__main__': block